libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/route/ReplayRouteMapperTest.kt

```diff
@@ -123,6 +128,27 @@ class ReplayRouteMapperTest {
         assertNull(replayLocation.location.speed)
     }
 
+    @Test
+    fun `an artificial driver drives with almost constant speed along a motorway`() {
```

Comment:
Will you move this test to `ReplayRouteInterpolatorTest`? That is where the interpolator speed tests are

Reply:
I copied speed checks to `ReplayRouteInterpolatorTest` to make sure that `createSpeedProfile` works in isolation. 

But I would like to keep current test as is. I like that it describes feature on the same level our customer use it. I think it's more important to make sure that the feature works for customer rather than a piece of logic which creates a speed profile works fine. What do you think about it? Are you okay with a bit of duplication causing by testing similar things in the context of different units? 🙂

Reply:
i usually start with a test in `ReplayRouteMapper`. but then change, move or copy the tests to another class. there have been many issues that are now described by a test.

It's possible that we make `ReplayRouteDriver` an interface dependency, at which point this would move. it could also be moved to `ReplayRouteDriver`, since it is describing a driver behavior. 

It sounds nice to say it belongs here because it's what the customer expects, but you could say that for everything. As you can see none of the traffic tests are in the mapper, they're in `ReplayRouteTrafficTest`.

It's no big deal though, I'm fine leaving it for now. And actually the `german_motorway_direction_route.json` may be useful for other tests. I'm looking more, and that file has traffic speeds. It seems like a good test to say, that the driver will not drive a constant speed on a highway with traffic.

Reply:
When I wrote tests for traffic, i didn't have many examples of routes with traffic. So this test fixture could help us find an issue